cdc/processor.go

```diff
@@ -363,7 +366,11 @@ func (p *processor) positionWorker(ctx context.Context) error {
 			}
 
 			p.position.ResolvedTs = minResolvedTs
-			resolvedTsGauge.Set(float64(oracle.ExtractPhysical(minResolvedTs)))
+			phyTs := oracle.ExtractPhysical(minResolvedTs)
+			resolvedTsGauge.Set(float64(phyTs))
+			// It is more accurate to get tso from PD, but in most cases we have
+			// deployed NTP service, a little bias is acceptable here.
+			metricResolvedTsLagGauge.Set(float64(oracle.GetPhysical(time.Now())-phyTs) / 1e3)
```

Comment:
how about get a ts from PD?

Reply:
As the comment in L371-L372, it is a trade-off. Get `tso` from PD may fail, timeout or increase the load of PD (although this is not too frequent).